protocol-specs/package-lock.json

```diff
@@ -5,22 +5,28 @@
 	"requires": true,
```

Comment:
Why did this file changed but no package.json? Was this outdated vs package.json?

Reply:
yes i think protocol-spec package json was outdated